docs/en/stack/ml/anomaly-detection/anomaly-detection-scale.asciidoc

```diff
@@ -0,0 +1,226 @@
+{anomaly-jobs} model memory API] manually. The estimation is based on the 
+analysis configuration details for the job and cardinality estimates for the 
+fields it references.
+
+
+[[model-snapshots]]
+== Setting model snapshot retention time 
+
+Model snapshots are created every 3 to 4 hours by default. Although the model 
+snapshots are compressed, they can still be extensive for large jobs. For 
+example, if you have a 30 GB job, the model snapshot can be 5 GB. Calculating 
+with the default values, six model snapshots are taken from the job per day. If 
+you keep them for ten days, it is 300 GB for a single job. By default, if there 
+are snapshots over one day older than the newest snapshot, they are deleted 
+except for the first snapshot each day. As well, all snapshots over ten days 
+older than the newest snapshot are deleted. There are two things to consider 
+about model snapshot retention:
+
+* how frequently you want to create snapshots from the model,
+* for how long you want to retain the snapshots.
+
+You can fine-tune the frequency of the model persistence and the snapshot 
+retention days by setting the following configuration options:
+
+* `background_persist_interval` sets the time between each periodic persistence 
+  of the model. The default value is a randomized value between 3 to 4 hours, 
+  which avoids all jobs persisting at the same time. Increase this value 
+  appropriately to fit your job.
+
+* `daily_model_snapshot_retention_after_days` affects the automatic removal of 
+  old model snapshots for the job. It specifies a period of time (in days) after 
+  which only the first snapshot per day is retained. This period is relative to 
+  the timestamp of the most recent snapshot for this job. Consider setting this 
+  value to only a couple of days after which one snapshot per day will be 
+  retained.
+
+* `model_snapshot_retention_days` defines the time in days that model snapshots 
+  are retained for the job. The default value is ten days meaning snapshots ten 
+  days older than the newest snapshot are deleted.
+
+For more information, refer to <<ml-model-snapshots>>.
+
+TIP: When your job is shut down, and it is not closed properly, it rolls back to 
+the last successfully saved snapshot when it is started again. Close the job 
+gracefully when it is possible instead of killing it so that you can preserve 
+the exact model state.
+
+
+[[results-retention]]
+== Setting result retention time
+
+{anomaly-detect-cap} results are retained indefinitely by default. For big jobs, 
+results build up over time, and your result index will be quite large. Consider 
+how long you wish to retain the results and set `results_retention_days` 
+accordingly – for example, to 30 or 60 days – to avoid unnecessarily large 
+result indices.   
+
+
+[[renormalization]]
+== Configuring renormalization window days
+
+When a new anomaly has a higher score than any anomaly in the past, the anomaly 
```

Comment:
This is technically not correct. We only do this when we have detected a "big change"